lookout/style/format/tests/code_generator_data.py

```diff
@@ -0,0 +1,472 @@
+from collections import OrderedDict
+
+from lookout.style.format.feature_utils import (
+    CLASS_INDEX, CLS_DOUBLE_QUOTE, CLS_NEWLINE, CLS_NOOP, CLS_SINGLE_QUOTE, CLS_SPACE,
+    CLS_SPACE_DEC, CLS_SPACE_INC)
+
+composite_to_label_names = [
+    (CLS_SPACE,),
+    (CLS_SPACE, CLS_SINGLE_QUOTE),
+    (CLS_SPACE, CLS_DOUBLE_QUOTE),
+    (CLS_NEWLINE, ),
+    (CLS_NEWLINE, CLS_NEWLINE),
+    (CLS_NEWLINE, CLS_SPACE_INC),
+    (CLS_NEWLINE, CLS_SPACE_DEC),
+    (CLS_NEWLINE, CLS_SPACE_INC, CLS_SPACE_INC),
+    (CLS_NEWLINE, CLS_SPACE_DEC, CLS_SPACE_DEC),
+    (CLS_NEWLINE, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC),
+    (CLS_NEWLINE, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC),
+    (CLS_NEWLINE, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC, CLS_SPACE_DEC,
+        CLS_SPACE_DEC),
+    (CLS_SPACE_INC, ),
+    (CLS_SPACE_DEC, ),
+    (CLS_SINGLE_QUOTE, ),
+    (CLS_DOUBLE_QUOTE, ),
+    (CLS_NOOP, ),
+]
+
+label_composites = [
+    tuple(CLASS_INDEX[cls] for cls in label_name)
+    for label_name in composite_to_label_names]
+
+labels_to_composite = {x: i for i, x in enumerate(composite_to_label_names)}
+
+original_code = """import { makeToast } from '../../common/app/Toasts/redux';
```

Comment:
I would recommend creating a directory where to dump contents of a file - right now it's super hard to read this test. And than just read content - and everybody could read specific file when it's needed

Reply:
I agree it is hard to read it, but it is data for the test and I put it to separate file.
I think about something like you suggest but there are several problems.
1. It is a lot of small files (minor)
2. It is JS code files, and I think we are avoiding JS code in our repo. So it will require to zip them and you can not read it in this case.

I can put comment separator between cases for better readability if you want.